en/components/grid_filtering.md

```diff
@@ -29,16 +29,16 @@ There's a default filtering strategy provided out of the box, as well as all the
 > [!NOTE]
 > If values of type `string` are used by column of dataType `Date`, the grid won't parse it to `Date` objects and using filtering conditions won't be possible. If you want to use `string` objects, additional logic should be implemented on the application level, in order to parse the values to `Date` object.
 
-You can filter any column or a combination of columns through the grid API. The grid exposes several methods for this task - `filter`, `filterGlobal ` and `clearFilter`.
+You can filter any column or a combination of columns through the grid API. The grid exposes several methods for this task - [`filter`](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxgridcomponent.html#filter), [`filterGlobal`](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxgridcomponent.html#filterglobal) and [`clearFilter`](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxgridcomponent.html#clearfilter).
 
-*   `filter` - filter a single column or a combination of columns.
+*   [`filter`]([`filter`](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxgridcomponent.html#filter)) - filter a single column or a combination of columns.
```

Comment:
There is no API Reference for the Filtering Topic.

Reply:
Added API References section